src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_template_builder.py

```diff
@@ -859,7 +859,7 @@ def build_vmss_resource(name, naming_prefix, location, tags, overprovision, upgr
 
 def build_av_set_resource(name, location, tags,
                           platform_update_domain_count, platform_fault_domain_count, unmanaged):
-    av_set_api_version = get_api_version(ResourceType.MGMT_COMPUTE)
+    av_set_api_version = get_api_version(ResourceType.MGMT_COMPUTE, 'availability_sets')
```

Comment:
@derekbekoe This is saying "For RP Compute, and RT AvailabilitySets. This is consistent with Profile, since profile have these two levels as well.
As you were saying, we can change the code to:
```python
av_set_api_version = get_api_version(ResourceType.MGMT_COMPUTE_AVAILIBILITY_SETS)
```
But in that case we must do that for ALL RT. Feels cumbersome for me, but up to you.

Reply:
@lmazuel, a quick question here, do we have to make this change? The old code should still work, right? I mean ` av_set_api_version = get_api_version(ResourceType.MGMT_COMPUTE)`

Reply:
The old code will get the default API version of Compute. Since now you can specify RT override (like you want for resource_skus), it's safer to be explicit about the RT you want. Remember, Azure Profiles are at the RT level, the current code is a **guess** that the APIVersion is available across RT.

Reply:
Another syntax already available:
```python
av_set_api_version = get_mgmt_service_client(ResourceType.MGMT_COMPUTE).availability_sets.api_version
```